### PR TITLE
Add Duration and QueuedDuration Job property

### PR DESCRIPTION
**Pull request by Toa741**

### CODE REVIEW

File: NGitLab.Mock/Job.cs [hunk 1]
```diff
@@ -46,6 +46,10 @@ public Job()
 
         public string WebUrl => Server.MakeUrl($"{Project.PathWithNamespace}/-/jobs/{Id.ToString(CultureInfo.InvariantCulture)}");
 
+        public decimal? Duration { get; set; }
```
Comment (meziantou) [inline NGitLab.Mock/Job.cs]:
```suggestion
        public float? Duration { get; set; }
```

File: NGitLab.Mock/Job.cs [hunk 2]
```diff
@@ -46,6 +46,10 @@ public Job()
...
+
+        public decimal? QueuedDuration { get; set; }
```
Comment (meziantou) [inline NGitLab.Mock/Job.cs]:
```suggestion
        public float? QueuedDuration { get; set; }
```

File: NGitLab.Mock/PublicAPI.Unshipped.txt [hunk 3]
```diff
@@ -482,6 +482,8 @@ NGitLab.Mock.Job.Coverage.get -> double?
 NGitLab.Mock.Job.Coverage.set -> void
 NGitLab.Mock.Job.CreatedAt.get -> System.DateTime
 NGitLab.Mock.Job.CreatedAt.set -> void
+NGitLab.Mock.Job.Duration.get -> decimal?
```
Comment (meziantou) [inline NGitLab.Mock/PublicAPI.Unshipped.txt]:
```suggestion
NGitLab.Mock.Job.Duration.get -> float?
```

File: NGitLab/Models/Job.cs [hunk 4]
```diff
@@ -60,6 +60,12 @@ public class Job
         [DataMember(Name = "web_url")]
         public string WebUrl;
 
+        [DataMember(Name = "duration")]
+        public decimal? Duration;
```
Comment (meziantou) [inline NGitLab/Models/Job.cs]:
```suggestion
        public float? Duration;
```

File: NGitLab/Models/Job.cs [hunk 5]
```diff
@@ -60,6 +60,12 @@ public class Job
...
+
+        [DataMember(Name = "queued_duration")]
+        public decimal? QueuedDuration;
```
Comment (meziantou) [inline NGitLab/Models/Job.cs]:
```suggestion
        public float? QueuedDuration;
```

**Review by meziantou — CHANGES_REQUESTED** [review]
I don't think you need to use `decimal`. `float` should be enough. If you have a doubt, you can use 
 `double`

File: NGitLab.Mock/PublicAPI.Unshipped.txt [hunk 6]
```diff
@@ -493,6 +495,8 @@ NGitLab.Mock.Job.Pipeline.get -> NGitLab.Models.Job.JobPipeline
 NGitLab.Mock.Job.Pipeline.set -> void
 NGitLab.Mock.Job.Project.get -> NGitLab.Models.Job.JobProject
 NGitLab.Mock.Job.Project.set -> void
+NGitLab.Mock.Job.QueuedDuration.get -> decimal?
```
Comment (Toa741) [inline NGitLab.Mock/PublicAPI.Unshipped.txt]:
```suggestion
NGitLab.Mock.Job.QueuedDuration.get -> float?
```

File: NGitLab/PublicAPI.Unshipped.txt [hunk 7]
```diff
@@ -1617,6 +1617,7 @@ NGitLab.Models.Job.Artifacts -> NGitLab.Models.Job.JobArtifact
 NGitLab.Models.Job.Commit -> NGitLab.Models.Commit
 NGitLab.Models.Job.Coverage -> double?
 NGitLab.Models.Job.CreatedAt -> System.DateTime
+NGitLab.Models.Job.Duration -> decimal?
```
Comment (Toa741) [inline NGitLab/PublicAPI.Unshipped.txt]:
```suggestion
NGitLab.Models.Job.Duration -> float?
```

File: NGitLab/PublicAPI.Unshipped.txt [hunk 8]
```diff
@@ -1645,6 +1646,7 @@ NGitLab.Models.Job.JobRunner.Name -> string
 NGitLab.Models.Job.Name -> string
 NGitLab.Models.Job.Pipeline -> NGitLab.Models.Job.JobPipeline
 NGitLab.Models.Job.Project -> NGitLab.Models.Job.JobProject
+NGitLab.Models.Job.QueuedDuration -> decimal?
```
Comment (Toa741) [inline NGitLab/PublicAPI.Unshipped.txt]:
```suggestion
NGitLab.Models.Job.QueuedDuration -> float?
```

**Review by meziantou — APPROVED** [review]

**Pull request merged**